data/items.ts

```diff
@@ -2312,6 +2327,21 @@ export const Items: {[itemid: string]: ItemData} = {
 		num: 286,
 		gen: 4,
 	},
+	griseouscore: {
+		name: "Griseous Core",
+		spritenum: 0, // TODO
```

Comment:
It's not accurate but maybe we could use the existing orb item sprites as placeholders?

Reply:
Yeah that seems fine. I'll leave the TODO comment so we can remember to change it.